browser/js/graphs/pie.js

```diff
@@ -13,7 +13,7 @@ app.directive('pieChart', function(d3Service, $window) {
 
```

Comment:
We'll have merge conflicts between this and the pie chart Corey and I worked on. Are we going to keep this or just use the one we had working yesterday?